webpack.config.js

```diff
@@ -25,10 +23,7 @@ const copy_files = [
 const plugins = [
     new copy({ patterns: copy_files }),
     new extract({filename: "[name].css"}),
-    new ESLintPlugin({
-        extensions: ["js", "jsx"],
-        failOnWarning: true,
```

Comment:
That seems useful though -- perhaps we should rather enable that in starter-kit as well?